src/legacy/state/orders/consts.ts

```diff
@@ -16,3 +18,8 @@ export const SPOT_PRICE_CHECK_POLL_INTERVAL = ms`15s`
 export const EXPIRED_ORDERS_CHECK_POLL_INTERVAL = ms`15s`
 
 export const OUT_OF_MARKET_PRICE_DELTA_PERCENTAGE = new Percent(1, 100) // 1/100 => 0.01 => 1%
+
+// Clear order's storage
+export const ORDER_STORAGE_KEY = 'redux_localstorage_simple_orders'
+export const ORDER_STATUSES_TO_CLEAR: OrderTypeKeys[] = ['expired', 'cancelled', 'fulfilled']
```

Comment:
`CONFIRMED_STATES` probably should replace it

Reply:
Ah good suggestion, updated!